source/process/release-process.md

```diff
@@ -268,23 +284,33 @@ If a bug fix release is required, run through the following steps:
 
 Once final dot release build is ready to cut:
 
-1. Build:  
+1. Dev:
+    - Verify hashes and GPG signatures are correct, once build is cut
+    - Delete RCs after final version is shipped
+    - Merge changes made to release branch into vXXX
+2. Build:  
     - Tag a new release (e.g. 1.1.1) and run an official build  
     - Update CI servers, pre-release and GitLab Mattermost to the final version    
-2. PM:
+3. PM:
     - Update [Mattermost pricing page](https://about.mattermost.com/pricing/) if anything has changed
     - Merge the Changelog PR with notes on patch releases (see [example entry](https://docs.mattermost.com/administration/changelog.html#release-v3-5.1))
     - Submit GitLab MR to update the version number and MD5 hash to the dot release version. [See example](https://gitlab.com/gitlab-org/omnibus-gitlab/merge_requests/1127)
-      - [Test the upgrade](https://docs.google.com/document/d/1mbeu2XXwCpbz3qz7y_6yDIYBToyY2nW0NFZq9Gdei1E/edit#heading=h.ncq9ltn04isg) once the MR is merged and the package is released to the GitLab package server
-3. QA:  
+    - Update the version archive in the [upgrade guide](https://github.com/mattermost/docs/blob/master/source/administration/upgrade.md)
+    - [Test the upgrade](https://docs.google.com/document/d/1mbeu2XXwCpbz3qz7y_6yDIYBToyY2nW0NFZq9Gdei1E/edit#heading=h.ncq9ltn04isg) once the MR is merged and the package is released to the GitLab package server
+4. QA:  
     - Verifies each of the issues in the patch release are fixed
-4. Docs:
+5. Docs:
     - Update the version archive in the [upgrade guide](https://github.com/mattermost/docs/blob/master/source/administration/upgrade.md)
     - Submit an MR to update [GitLab Mattermost documentation](https://docs.gitlab.com/omnibus/gitlab-mattermost/README.html)
-5. Logistics:
+6. Logistics:
     - Update [Mattermost server download page](https://mattermost.org/download) with the links to the EE and TE bits
       - Test the download links before and after updating the page
-6. Marketing:
+    - Contact owners of [community installers](http://www.mattermost.org/installation/) or submit PRs to update install version number
+    - For Puppet, Heroku and Ansible Playbook, post to Installers and Images channel announcing the new release.
+    - For Chef Cookbook, open a new issue to announce the new release.
```

Comment:
Add two more lines of indentation so they're sub-bullets of `Contact owners of ...`. Same for Yunohost and OpenShift

Reply:
Also, can you grab the links included in T-0 checklist? There are a few examples linked for reference: https://github.com/mattermost/docs/blob/dccf74880f859e2ff3b32cdcaee318e7ced8f9a7/source/process/release-process.md#i-t-minus-0-working-days-release-day

Reply:
Done